content/en/docs/user-documentation/paragraphs/secondary-description/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Secondary Description and Sidebar
+description: Add a bottom area element with two columns of reusable content for anchoring a page.
+---
+
+> Note: This element does not work properly in Carnation.
```

Comment:
do we have a ticket for it? @froboy

Reply:
@podarok no ticket, but as this comment has existed since around 2020 when David Wells wrote it, I think it's probably safe to consider it outdated. We can flag it for deprecation?

```suggestion
> Note: This element does not work properly in Carnation and is considered deprecated.
```